MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -121,7 +121,7 @@ class RouteManeuverViewController: UIViewController {
         
         if routeProgress.currentLegProgress.alertUserLevel == .arrive {
             distance = nil
-            destinationLabel.unabridgedText = routeStepFormatter.string(for: routeStepFormatter.string(for: routeProgress.currentLegProgress.upComingStep))
+            destinationLabel.unabridgedText = routeProgress.remainingWaypoints.last?.name ?? routeStepFormatter.string(for: routeProgress.currentLegProgress.upComingStep)
```

Comment:
Let’s use `routeProgress.currentLeg.destination.name` instead; that makes things a little clearer and prepares this method for when #270 lands.